apps/nuxt-components/stories/BackgroundGradient.story.vue

```diff
@@ -0,0 +1,7 @@
+<template>
+  <Story :layout="{ type: 'grid', width: '75%' }">
```

Comment:
i really don't understand how changing the layout solved the base url issue ,

Reply:
it is a bug in histoire